hivemind_core/scripts.py

```diff
@@ -157,6 +157,15 @@ def list_clients():
 
 
 @hmcore_cmds.command(help="start listening for HiveMind connections", name="listen")
+@click.option(
+    "--remote_bus_host",
+    help="Remote bus address",
+    type=str,
+    default="127.0.0.1",
+)
+@click.option(
+    "--remote_bus_port", help="Remote bus port number", type=int, default=8181
```

Comment:
can this be called ovos-bus-address? make it clear its not a hivemind bus since both use same nomenclature